go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/atreugo/mock v0.0.0-20200601091009-13c275b330b0
 	github.com/fasthttp/router v1.4.16
+	github.com/json-iterator/go v1.1.12
```

Comment:
Hi @boatrainlsz,

Thank you so much for your contribution and for using atreugo!

The PR looks good, but i wouldn't want to add a new dependency for json marshaling (in this case). 
So i suggest you just create an interface for marhshaling, add it to config type and create a default function that does the same as before your changes! 

In this way, it's more configurable and extensible to any marshaler! 😉

Reply:
> Hi @boatrainlsz,
> 
> Thank you so much for your contribution and for using atreugo!
> 
> The PR looks good, but i wouldn't want to add a new dependency for json marshaling (in this case). So i suggest you just create an interface for marhshaling, add it to config type and create a default function that does the same as before your changes!
> 
> In this way, it's more configurable and extensible to any marshaler! 😉

@savsgio Thanks for your advice! I'll take a look.